app/src/main/AndroidManifest.xml

```diff
@@ -10,53 +10,55 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity
+            android:name=".view.edit.EditQuizActivity"
+            android:label="@string/title_activity_edit_quiz"
+            android:parentActivityName=".view.home.HomeActivity">
+
+            <intent-filter>
+                <action android:name="android.intent.action.EDIT" />
 
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
         <activity
             android:name=".view.LoginActivity"
             android:label="@string/title_activity_login">
-
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-
         <activity
             android:name=".view.home.HomeActivity"
-            android:parentActivityName=".view.LoginActivity"
```

Comment:
To be honest, I don't know how I can create the "up" button in another way. If you know, feel free to tell me... By the way I filled the android manifest quite quickly because the main goal of this PR was to create the list edit view, not to actually integrate it in the app.

Reply:
Actually, I know what happened here. I deleted the meta-data tags because they're intended for API level 15 and lower. Since we are using API level 21, we don't need them. And the line that you highlighted didn't disappear, it was just moved a few lines afterwards (blame on `git diff` -_- ).